package.json

```diff
@@ -41,6 +41,8 @@
     "preq": "^0.3.13",
     "service-runner": "^0.1.8",
     "underscore": "^1.8.3",
+    "request": "2.57.*",
```

Comment:
Lets use preq instead - same as i used in graphoid - https://github.com/wikimedia/mediawiki-services-graphoid/blob/master/routes/graphoid-v1.js  -- this way we keep all async stuff uniform with promises